assembl/graphql/tag.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding=utf-8 -*-
+import graphene
+from graphene.relay import Node
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+import assembl.graphql.docstrings as docs
+from assembl.auth import CrudPermissions
+from assembl import models
+
+from .permissions_helpers import require_instance_permission
+from .types import SecureObjectType, SQLAlchemyInterface
+from .utils import abort_transaction_on_exception
+
+
+class TagInterface(SQLAlchemyInterface):
+    __doc__ = docs.TagInterface.__doc__
+
+    class Meta:
+        model = models.Keyword
+        only_fields = ('value', )
+        # Don't add id in only_fields in an interface
+        # will be just the primary key, not the base64 type:id
+
+    value = graphene.String(description=docs.TagInterface.value)
```

Comment:
You should maybe add `required=True`?
You don't need to list `value` in `only_fields` if you write `value = ...`.